src/shared/algoliaUserAgent.ts

```diff
@@ -0,0 +1,6 @@
+import { createUserAgent } from "@algolia/transporter";
+import algoliasearch from "algoliasearch";
+import packageJson from "../../package.json";
+
+export const customUserAgent = createUserAgent(algoliasearch.version)
+  .add({segment: 'Kontent.ai', version: packageJson.version});
```

Comment:
Do we want to use the name property from `package.json`? If we do I might vote for using  `@kontent-ai/example-integration-algolia`.

Reply:
adjusted and used